.travis/build_dec.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+$SCRIPT_BEGIN
+
+### Dec
+echo "Compile Dec in non parallel mode to save memory (to fix gcc internal compiler error(Killed))";
+
+cd "$BUILD_DIR"
+make exampleDiscreteExteriorCalculusChladni
+#make exampleDiscreteExteriorCalculusSolve
+#make exampleDECSurface
```

Comment:
why these one have been commented ?

Reply:
It is a copy-paste from `.travis/main_build.sh`:
https://github.com/DGtal-team/DGtal/blob/8f3ca8c0ddae25e1be2b745fbb1c039e494b0b91/.travis/main_build.sh#L16
and it has been commented in 163a8357c6d4e2380c0fd4e2a0c21093c7a08e0f from #1306 but I don't know why (@kerautret  ?).

I will try to uncomment these lines :octocat:

Reply:
It was to try to avoid the segfault or timeout issue of travis due to gcc problem (as discussed with Jaco in last meeting). Changing the moment where it was compiled fixed the memory issue (I suppose), but perhaps we have to remove this example from the travis test.

Reply:
In fact, these two examples are already built during the main build. This part is only for examples that need to be build alone in order to avoid compiler failures. I think we can remove these two lines ...

Reply:
Oups, didn't see your answer :wink:

Reply:
yes of course if they are in the main part, but it was commented because it was an alternative.
But this part is not robust in all cases.

Reply:
I was wondering ;)